README.md

```diff
@@ -197,6 +197,63 @@ class Serializer(serializers.Serializer):
 The implementation of `inline_serializer` can be found in `utils.py` in this repo.
 
 
+## Exception Handling
+
+### Raising Exceptions in Services
+Now we have a great separation between business logic and HTTP logic. The business logic lives in the services and the HTTP logic lives in the APIs.
+
+In order to keep this separation of concerns our services must not use the `rest_framework.exception` classes because they are bounded with HTTP status codes. 
+Our services must use: `native python exceptions`, `django.core.exceptions` or some custom business exceptions that we define.
+
+Here is a good example of service that preforms some business validation and raise `django.core.exceptions.ValidationError`
+```python
+from django.core.exceptions import ValidationError
+
+def create_topic(*, name: str, course: Course) -> Topic:
+    if Topic.objects.filter(course=course, name=name).exists():
```

Comment:
This looks like a `unique_together` validation :thinking: Perhaps we need a better example